jenkins-ansible/README.md

```diff
@@ -0,0 +1,194 @@
+# 
+# PetPortal Ansible-Jenkins
+
+This is my Pet Portal which uses Ansible to provision the targeted environemnt and Jenkins to Build and Test the
+ applicaion.  This blueprint also leverages the xlr-xlrelease-plugin to dynamically tag the releases with the 
+ Service Now tickets.
+
+## Setup Steps
+1. Clone this repository
+2. Configure your ~/.xebialabs/config.yaml for your connection to the XL platform tools
+```
+    xl-deploy:
+      username: admin
+      password: admin
+      url: http://localhost:4516/
+    xl-release:
+      username: admin
+      password: admin
+      url: http://localhost:5516/
+    blueprint-repository:
+      username: ""
+      password: ""
+      # file:// protocol is not supportted so you'll need to run a local http server
+      # E.g. in blueprint directory run: python -m SimpleHTTPServer 3516
+      url: ttp://localhost:3516
+```
+_3. Create the Blueprints
```

Comment:
Please remove this as well as it doesnt serve a purpose as user will see this only after performing these steps

Reply:
removed